src/utils/photoFrame/index.ts

```diff
@@ -13,3 +13,14 @@ export async function isPlaybackPossible(url: string): Promise<boolean> {
         video.src = url;
     });
 }
+
+export const livePhotoBtnHTML = `
```

Comment:
we should keep `html` code in the component directory

you can create a file `component/livePhotoBtn` and put it there